pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "usn"
+version = "0.1.0"
+description = "This utility generates a netboot directory tree from an Ubuntu Server Live ISO image, an image based on the subiquity installer."
+authors = ["Taihsiang Ho (tai271828) <taihsiang.ho@canonical.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.6.5"
```

Comment:
That rules out xenial, but I suppose that's OK since xenial has now entered extended support. And, it turns out that this script doesn't work in xenial today anyway:
```
$ ./ubuntu-server-netboot.py --url http://releases.ubuntu.com/focal/ubuntu-20.04.2-live-server-amd64.iso
INFO: Downloading http://releases.ubuntu.com/focal/ubuntu-20.04.2-live-server-amd64.iso
Traceback (most recent call last):
  File "./ubuntu-server-netboot.py", line 282, in <module>
    iso = ServerLiveIso(iso.name)
  File "./ubuntu-server-netboot.py", line 75, in __init__
    if self.version != udi.version(codename):
  File "./ubuntu-server-netboot.py", line 44, in version
    for release in self._releases:
AttributeError: 'UbuntuDistroInfoWithVersionSupport' object has no attribute '_releases'
```

If we choose to support xenial, we can loosen the python3 dep at the same time.

Reply:
Right. Let me know if we would like to support Xenial. I don't mind using python 3.5 (which is the default python3 in xenial) because the tool `ubuntu-server-netboot` does not rely on newer python features.

Additionally, I picked up python `3.6` on purpose to make sure the tool `ubuntu-server-netboot` supports Bionic.